vivisect/analysis/elf/elfplt.py

```diff
@@ -373,7 +373,7 @@ def analyzeFunction(vw, funcva):
                     taintrepr = emu.reprVivTaint(taintval)
                 else:
                     taintrepr = 'None'
-                logger.exception('0x%x: opval=0x%x: brflags is BR_DEREF, but loctup is None.  Don\'t know what to do. skipping.  taint:%r (%r)', op.va, opval, taintval, taintrepr)
+                logger.error('0x%x: opval=0x%x: brflags is BR_DEREF, but loctup is None.  Don\'t know what to do. skipping.  taint:%r (%r)', op.va, opval, taintval, taintrepr)
```

Comment:
I was wondering why we were getting print outs that just said `None` with none of the standard logging formatting in a few cases.